packages/nube-devtools/src/zip.js

```
@@ -0,0 +1,10 @@
+import gulp from 'gulp'
+import zip from 'gulp-zip'
+import { createRequire } from 'module'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)